js/initcontext.go

```diff
@@ -317,6 +313,40 @@ func (i *InitContext) Open(ctx context.Context, filename string, args ...string)
 	return i.runtime.ToValue(string(data)), nil
 }
 
+func readFile(fileSystem afero.Fs, filename string) ([]byte, error) {
+	// Workaround for https://github.com/spf13/afero/issues/201
+	if isDir, err := afero.IsDir(fileSystem, filename); err != nil {
```

Comment:
Because of this the following code:
```
if (__VU == 1) {
   open("./somenonexistingfile.js")
}
```

errors out with "no such file or directory" instead of the message below. This can be fixed later, as at least I don't know of a good fix without having to reimplement `fs.Stat` as well and we want to [move away](#1079 ) from afero either way. On the other hand that doesn't seem like that big of a change so :shrug:

Reply:
That's fine. It's a tiny change :+1: